library/src/main/java/com/xfinity/blueprint/view/ScreenViewDelegate.kt

```diff
@@ -92,7 +92,9 @@ open class ScreenViewDelegate(componentRegistry: ComponentRegistry,
      * @param newComponents A List of the components the screen should be updated to display
      */
     override fun updateComponents(newComponents: List<Component>) {
-        updateComponents(newComponents, { oldComponents -> DefaultComponentDiffCallback(oldComponents, newComponents) })
+        viewRegistry.updateComponents(newComponents) {
```

Comment:
I feel the chained calls like this are problematic. In the view registry, you are not actually updating anything.

Reply:
Updates are happening. Can you clarify?